docs/pbc/all/payment-service-provider/202404.0/base-shop/third-party-integrations/payone/integration-in-the-back-office/integrate-payone.md

```diff
@@ -250,113 +307,36 @@ class RouterDependencyProvider extends SprykerRouterDependencyProvider
-    xmlns="spryker:oms-01"
-    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="spryker:oms-01 http://static.spryker.com/oms-01.xsd"
->
-
-    <process name="SomePaymentProcess" main="true">
-
-        <!-- other configurations -->
-
-        <states>
-
-            <!-- other states -->
-
-          <state name="payment capture pending" display="oms.state.in-progress"/>
-
-            <!-- other states -->
-
-        </states>
-
-        <transitions>
-
-            <!-- other transitions -->
-
-            <transition happy="true">
-              <source>ready for dispatch</source>
-              <target>payment capture pending</target>
-              <event>capture payment</event>
-            </transition>
-
-            <!-- other transitions -->
-
-        </transitions>
-
-        <events>
-
-            <!-- other events -->
-
-            <event name="capture payment" onEnter="true" command="Payment/Capture"/>
-
-            <!-- other events -->
-
-        </events>
-
-    </process>
-
-</statemachine>
-```
-
-[Read more](/docs/acp/user/acp-payment-oms-guides.html) about ACP payment methods integration with your project OMS configuration.
-
-### 9. Accommodate data and presentation change
-
-The newly added glossary keys require a few additional actions. Do the following:
-
-1. Add the following glossary keys to your glossary data import file:
-
-```csv
-...
-payment_foreign.thank_you,Danke,de_DE
-payment_foreign.thank_you,Thank you,en_US
-payment_foreign.go_to_my_account,Gehe zum "Benutzerkonto",de_DE
-payment_foreign.go_to_my_account,Go to "My Account",en_US
-payment_foreign.success.order.placement.message,Deine Bestellung ist erfolgreich bei uns eingegangen. Die Bestellbestätigung wurde soeben per E-Mail versendet. In deinem Benutzerkonto kannst deine Bestellung überprüfen und nachverfolgen.,de_DE
-payment_foreign.success.order.placement.message,Your order has been paid successfully. You will receive your order confirmation email in a few minutes. You can check and track your order in your account.,en_US
-payment.cancellation.title,Bezahlvorgang abgebrochen,de_DE
-payment.cancellation.title,Payment cancellation,en_US
-payment.cancellation.message,Du hast den Bezahlvorgang abgebrochen.,de_DE
-payment.cancellation.message,You have cancelled your payment.,en_US
-oms.state.reservation-cancelled,Payment cancelled,en_US
-oms.state.reservation-cancelled,Payment cancelled,de_DE
-oms.state.reservation-cancellation-pending,Payment cancellation in progress,en_US
-oms.state.reservation-cancellation-pending,Payment cancellation in progress,de_DE
-```
-2. Run the data import for the glossary:
-
-```bash
-console data:import glossary
-```
+9. [Integrate Payone into OMS](/docs/dg/dev/acp/integrate-acp-payment-apps-with-spryker-oms-configuration.html#configuring-oms-for-your-project).
 
-### 10. Optional: Introduce template changes in `CheckoutPage`
+### Optional: Introduce template changes in `CheckoutPage`
```

Comment:
I think it's not option if the next text says 
`If you rewrote ...` so if has to be done in case these files are changed on project level